Misc/NEWS.d/next/Core and Builtins/2020-10-31-16-54-00.bpo-42217.GdcHe5.rst

```diff
@@ -0,0 +1 @@
+Make the compiler merges co_code and co_lnotab in a module.
```

Comment:
```suggestion
Make the compiler merge module attributes co_code and co_lnotab.
```

Reply:
It is not module attributes.

There are many code objects in module. All code modules have co_code and co_lnotab that are bytes objects.
Since bytes objects are immutables, same objects can be shared.

Reply:
How about "Make the compiler merges same co_code and co_linetable in a module like already did for co_consts."

Reply:
Ah, then I don’t understand this well enough (the PR or the suggested text) to say 😐 

When you pick a phrasing, I think it will be necessary to update the PR title + fix the extra s (should be `make compiler merge`, not merges).